static/app/views/replays/detail/layout.tsx

```diff
@@ -0,0 +1,272 @@
+          {content}
+        </Container>
+      );
+    default:
+      return <div>TODO</div>;
+  }
+}
+
+const Container = styled('div')`
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+  display: flex;
+  flex-flow: nowrap column;
+  overflow: hidden;
+  padding: ${space(2)};
+
+  .resizeWidthBar {
+    cursor: ew-resize;
+    height: 100%;
+    width: ${space(2)};
+  }
+
+  .resizeHeightBar {
+    cursor: ns-resize;
+    height: ${space(2)};
+    width: 100%;
+  }
+  .resizeHeightBar.overlapDown {
+    height: calc(16px + 34px); /* Spacing between components + height of <FocusTabs> */
+    margin-bottom: -34px; /* The height of the <FocusTabs> text + border */
+    z-index: ${p => p.theme.zIndex.initial};
+  }
+
+  .resizeWidthBar,
+  .resizeHeightBar {
+    background: transparent;
+    display: flex;
+    align-items: center;
+    align-content: center;
+    justify-content: center;
+  }
+  .resizeWidthBar:hover,
+  .resizeHeightBar:hover {
+    background: ${p => p.theme.hover};
+  }
+
+  .resizeWidthHandle {
+    height: ${space(3)};
+    width: ${space(2)};
+  }
+
+  .resizeHeightHandle {
+    height: ${space(2)};
+    width: ${space(3)};
+    transform: rotate(90deg);
+  }
+
+  .resizeWidthHandle > span,
+  .resizeHeightHandle > span {
+    display: block;
+    background: transparent ${GrabberSVG} center center no-repeat;
+    width: 100%;
+    height: 100%;
+  }
+`;
+
+type ResizePanelProps = {
+  direction: 'n' | 'e' | 's' | 'w';
+  minHeight?: number;
+  minWidth?: number;
+  modifierClass?: string;
+};
+
+const ResizePanel = styled(
+  ({direction, modifierClass = '', ...props}: ResizePanelProps) => {
+    const movesUpDown = ['n', 's'].includes(direction);
+    const borderClass = movesUpDown ? 'resizeHeightBar' : 'resizeWidthBar';
+    const handleClass = movesUpDown ? 'resizeHeightHandle' : 'resizeWidthHandle';
+
+    return (
+      <BaseResizePanel
+        direction={direction}
+        {...props}
+        borderClass={`${borderClass} ${modifierClass}`}
+        handleClass={`${handleClass} ${modifierClass}`}
+      />
+    );
+  }
+)`
+  position: relative;
+`;
+
+const PageColumn = styled('section')`
+  display: flex;
+  flex-grow: 1;
+  flex-wrap: nowrap;
+  flex-direction: column;
+`;
+
+const PageRow = styled(PageColumn)`
+  flex-direction: row;
+`;
+
+const TimelineSection = styled(PageColumn)`
+  flex-grow: 0;
+`;
+
+const ContentSection = styled(PageColumn)`
+  flex-grow: 3; /* Higher growth than SidebarSection or TopVideoSection */
+
+  height: 100%;
+  min-height: 300px;
+  width: 100%;
+`;
+
+const VideoSection = styled(PageColumn)`
+  flex-grow: 2;
+`;
+
+const BreadcrumbSection = styled(PageColumn)``;
+
+const SidebarSection = styled(PageColumn)`
+  min-width: ${SIDEBAR_MIN_WIDTH}px;
+`;
+
+const TopbarSection = styled(PageRow)`
+  height: 325px;
+  min-height: 325px;
+
+  ${BreadcrumbSection} {
+    max-width: 325px;
+    margin-left: ${space(2)};
+  }
+`;
+
+export default ReplayLayout;
```

Comment:
I would maybe consider splitting up this file a bit, e.g. Container + ResizePanel